app/controllers/api/braintree_controller.rb

```diff
@@ -1,31 +1,57 @@
 class Api::BraintreeController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
 
   def token
     render json: { token: ::Braintree::ClientToken.generate }
   end
 
   def transaction
-    sale = braintree.make_transaction(options)
+    result = braintree::Transaction.make_transaction(transaction_options)
+    if result.success?
+      render json: { success: true, transaction_id: result.transaction.id }
+    else
+      render json: { success: false, errors: result.errors }
```

Comment:
@NealJMD wouldn't you get more out of the error codes and messages than HTTP codes? I feel like the HTTP codes are more informative for the back end. But we can pass them for you if you'd like.